src/plugins/intel_cpu/src/nodes/executors/acl/acl_eltwise.cpp

```diff
@@ -51,7 +51,7 @@ bool AclEltwiseExecutorBuilder::isSupported(const EltwiseAttrs& eltwiseAttrs,
 //            case Algorithm::EltwisePowerDynamic: // TODO: ACL version doesn't work https://github.com/ARM-software/ComputeLibrary/issues/1047
         case Algorithm::EltwiseSoftRelu:
         case Algorithm::EltwiseClamp:
-        //case Algorithm::EltwiseSwish: // TODO: efficientdet-d0 accuracy drops if ACL Swish is used
+        case Algorithm::EltwiseSwish: // TODO: efficientdet-d0 accuracy drops if ACL Swish is used
```

Comment:
Thats better to add the ticket number instead of just commet

Reply:
@allnes please add CVS-109354

Reply:
@dmitry-gorokhov @alvoron done